### PR TITLE
Node 18+ updates #21

### PR DESCRIPTION
**Pull request by fajar-apri-alaska**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #248, #249, #250, #251 

## Summary:
#250 should be resolved already without more changes

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

## Type of change:

Please delete options that are not relevant.

- [x] Revision of an existing capability

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  - type: textarea
+    id: details
+    attributes:
+      label: Please describe the bug
+      description: List out the steps to reproduce the behavior
+      placeholder: |
+	        1. Go to '...'
+	        2. Click on '....'
+	        3. Scroll down to '....'
+	        4. See error
```
Comment (blackfalcon) [inline .github/ISSUE_TEMPLATE/bug_report.yml:31]: Too many spaces/tabs

File: .github/settings.yml [hunk 2]
```diff
@@ -59,71 +60,104 @@ branches:
...
 # Labels: define labels for Issues and Pull Requests
 labels:
-  - name: auro-input
-    cat: default
-    color: fa23e4
-    description:
-    process: Key filter for this repo
+  - name: duplicate
+    color: '156fad'
+    description: This Issue or Pull Request already exists
```
Comment (blackfalcon) [inline .github/settings.yml:73]:
```suggestion
  - name: auro-input
    color: 'fa23e4'
    description:
    process: Key filter for this repo
  - name: duplicate
    color: '156fad'
    description: This Issue or Pull Request already exists
```

File: .github/workflows/testPublish.yml [hunk 3]
```diff
@@ -24,7 +24,12 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
       - run: npm ci
-      - run: npm run build
+      - run: npm run build:sass:component
+      - run: npm run postCss:component
+      - run: npm run sass:render
+      - run: npm run dist:js
+      - run: npm run bundler
+      - run: npm test
```
Comment (blackfalcon) [inline .github/workflows/testPublish.yml:32]:
```suggestion
      - run: npm run build
```

File: .github/workflows/testPublish.yml [hunk 4]
```diff
@@ -39,7 +44,11 @@ jobs:
         with:
           node-version: 18
       - run: npm ci
-      - run: npm run build
+      - run: npm run build:sass:component
+      - run: npm run postCss:component
+      - run: npm run sass:render
+      - run: npm run dist:js
+      - run: npm run bundler
```
Comment (blackfalcon) [inline .github/workflows/testPublish.yml:51]:
```suggestion
      - run: npm run build
```

Comment (blackfalcon) [inline .github/workflows/testPublish.yml:51]:
```suggestion
      - run: npm run build
```

File: package.json [hunk 5]
```diff
@@ -19,71 +19,70 @@
...
     "postcss-selector-replace": "^1.0.2",
     "prismjs": "^1.29.0",
-    "rollup": "^3.20.0",
+    "rollup": "^3.21.5",
     "rollup-plugin-serve": "^2.0.2",
-    "sass": "^1.61.0",
-    "semantic-release": "^20.1.3",
-    "sinon": "^15.0.3",
-    "stylelint": "^15.4.0",
+    "sass": "^1.62.1",
```
Comment (blackfalcon) [inline package.json:72]: add

https://github.com/AlaskaAirlines/WC-Generator/blob/master/template/package.json#L75-L79

File: scripts/postCss.mjs [hunk 6]
```diff
@@ -1,5 +1,10 @@
+import chalk from 'chalk';
+import { createRequire } from 'node:module';
+import * as url from 'url';
+
+const require = createRequire(import.meta.url);
+
```
Comment (blackfalcon) [inline scripts/postCss.mjs:6]: Please update file to be like the following 

https://github.com/AlaskaAirlines/WC-Generator/blob/master/template/scripts/postCss.js

File: scripts/pre-commit.mjs [hunk 7]
```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const chalk = require('chalk');
+import chalk from 'chalk';
```
Comment (blackfalcon) [inline scripts/pre-commit.mjs:3]: Update file to be like the following 

https://github.com/AlaskaAirlines/WC-Generator/blob/master/template/scripts/pre-commit.mjs

**Review by blackfalcon — CHANGES_REQUESTED** [review]

**Comment by blackfalcon** [comment]
@fajar-apri-alaska please see new issue for missing updates

https://github.com/AlaskaAirlines/auro-input/issues/254

**Review by blackfalcon — CHANGES_REQUESTED** [review]
See additional comments.

**Comment by fajar-apri-alaska** [comment]
> @fajar-apri-alaska please see new issue for missing updates
> 
> * [Missing updates for Node 18+ support  #254](https://github.com/AlaskaAirlines/auro-input/issues/254)

Thanks for addressing the missing updates ~

**Review by blackfalcon — APPROVED** [review]

**Pull request merged**